src/shaders/circle.vertex.glsl

```diff
@@ -49,7 +62,14 @@ float circle_elevation(vec2 pos) {
 vec4 project_vertex(vec2 extrusion, vec4 world_center, vec4 projected_center, float radius, float stroke_width,  float view_scale) {
     vec2 sample_offset = calc_offset(extrusion, radius, stroke_width, view_scale);
 #ifdef PITCH_WITH_MAP
+#ifdef PROJECTION_GLOBE_VIEW
```

Comment:
The number of `ifdef` in this file are becoming a bit hard to reason about, I suggest to follow some of these patterns:
- Annotate the end of the `#ifdef` block by the define we're looking at with a comment, e.g.:
```
#ifdef SHADER_VARIANT
#endif // SHADER_VARIANT
```
- Indent based on `#ifdef` level, e.g.:
```
#ifdef SHADER_VARIANT_0
  #ifdef SHADER_VARIANT_1
  #endif
#endif
```
- A combination of both if we want to be extra explicit

Reply:
Good idea! I'd prefer the first version because in most places we don't indent ifdefs.

Reply:
Although never mind, it seems in this file we use indentation for the existing ifdefs, so I'll got with the second option.